code/index.html

```diff
@@ -4,19 +4,105 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+  
+</head>
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+    <section class="navigation-bar">
+    <img src="https://images.unsplash.com/photo-1573752040309-f4960b47735a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80" 
+      alt="matchsticksnews-logo"
+      class="matchsticksnews-logo"/>
+    <h1 class="site-name">Matchsticks News</h1>
+    
+    <a href="#home">Home</a>
+    <a href="#articles">Articles</a>
+    <a href="#home">Contact</a>
+
+    <div id="home"></div>
+
+    <div class="icon-box">
+    <a href="https://www.facebook.com/">
+    <img class="icon1" src="./social-media-icons/facebook_icon.png"/></a>
+    <a href="https://www.instagram.com/">
+    <img class="icon2" src="./social-media-icons/instagram_icon.png"/></a>
+    <a href="https://www.youtube.com/">
+    <img class="icon3" src="./social-media-icons/youtube_icon.png"/></a>
+    </section> 
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+    <section class="big-news"> 
+    <img src="https://www.guinnessworldrecords.com/assets/525970?ezimgfmt=ng%3Awebp%2Fngcb1%2Frs%3Adevice%2Frscb1-1&height=497&width=780"
+      alt="match-tower-master"/>  
+
+    <article class="big-news-article">  
+    <h2>Time to beat this record - Most matchsticks stacked into a tower in one minute!</h2>
+    <p>The most matchsticks stacked into a tower in one minute is 74, achieved by Silvio Sabba (Italy), in Pioltello, Italy, on 2 July 2012.
+        This record was broken on the Guinness World Record Challengers site.</p>
+    <p>Read all about this on<a href="https://www.guinnessworldrecords.com/world-records/most-matchsticks-stacked-into-a-tower-in-1-minute">Guiness World Record site!</a></p>
+
+    <p>Also check out these inspiring records of <a href="https://www.guinnessworldrecords.com/world-records/111405-most-matchsticks-extinguished-with-the-tongue-in-one-minute">Most matchsticks extinguished with the tongue in one minute</a>
+      and <a href="https://www.guinnessworldrecords.com/world-records/77623-match-throwing-distance">Farthest distance to throw a matchstick</a>.</p>
+    </article>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <hr class="solid-line">
+
+    <div id="articles"></div>
+
+    <section class="other-news">
+    <a href="#">
+    <img src="https://images.unsplash.com/photo-1612490689975-c062b1bdcec7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80"
+      alt="a paperheart taped on a matchstick"/>
+    <span><h3>Valentine's Day special: cute and fun crafting.</h3></span>
+    <div class="hidden-hover">Because who doesn't love presents? 🎁🧡</div></a>
```

Comment:
Very cool effect with the hover in the news! I really like the orange styling of them as well.